server/landing_page.go

```diff
@@ -413,8 +414,41 @@ func getSimilarPlaces(ctx context.Context, s *Server, dcid string) (
 	return result, nil
 }
 
+// Get nearby places.
```

Comment:
the test response seems to be missing place names for nearby places

Reply:
Right. The response data is not consistent now. Some of them does not have names as the client now does not need them. Added a todo to get all name information here as well so client does not bother to fetch the name and even parent abbreviations.